integration-tests/tests/utils/aws/waitForDownloadHash.ts

```diff
@@ -29,6 +29,7 @@ const retrieveSecureDownloadDbRecord = async (
   return result
 }
 
+// does this get used at all now? Safe to remove if not?
 export const waitForDownloadHash = async (
```

Comment:
I don't think we need this function anymore, as we are now calling notify instead of waiting for the download hash.

Reply:
thanks, deleted this